rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -122023,13 +122061,13 @@
 			"Monster": {
 				"Acheron mammoth": 4
 			},
-			"Nickname": "South-West Acheron Mammoths"
+			"Nickname": "South-West Acheron mammoth"
```

Comment:
Name of a chunk. Prolly shouldn't just refer to the chunk as "South-West mammoth" (singular)
The whole place is called the [Mammoth Iceberg](https://runescape.wiki/w/Mammoth_iceberg). Might make more sense to rename all fout chunks to "X Mammoth Iceberg" instead.